php-checks/src/main/java/org/sonar/php/checks/CheckList.java

```diff
@@ -334,7 +335,11 @@ private CheckList() {
     );
   }
 
+  public static List<Class<?>> getPhpChecks() {
+    return ListUtils.merge(getGeneralChecks(), getRegexChecks());
```

Comment:
Why can't we simply concatenate the 2 lists?

Reply:
Because of the list properties, the same object can appear twice. There are checks for common PHP files and for PHP ini files. So a concatenated list of both would contain these rules twice.